ietf-layer0-types-ext.yang

```diff
@@ -277,21 +287,21 @@ module ietf-layer0-types-ext {
                    mode.";
                 leaf-list supported-application-codes {
                   type leafref {
-                    path "../../mode-id";
+                    path "../../../mode-id";
                   }
-                  must "../../../supported-mode[mode-id=current()]/"
+                  must "../../../../supported-mode[mode-id=current()]/"

```

Comment:
This is correct but please check that we do not exceed the 69 line width:

```suggestion
                  must "../../../../"
                     + "supported-mode[mode-id=current()]/"
```

Co-authored-by: sergiobelotti sergio.belotti@nokia.com

Reply:
Addressed with https://github.com/ietf-ccamp-wg/ietf-ccamp-layer0-types-ext/pull/11/commits/9e2c4758a623ed7879d84d055d58807d3df2bc51